lib/components/project/screen/projectscreen.dart

```diff
@@ -348,7 +348,9 @@ class ProjectScreenState extends State<ProjectScreen> {
                                                       borderRadius:
                                                           BorderRadius.circular(
                                                               10),
-                                                      color: Color(0xffE4C864),
+                                                      color: getPriorityColor(
+                                                          state.projects[i]
```

Comment:
This code is part of a different bug fix which was related to how we display the difficulty tags on the dashboard screen. I just added here to save me some time lol